buildroot/rebuild.sh

```diff
@@ -3,7 +3,16 @@
 
 function download_br() {
     mkdir -p src
-    curl https://buildroot.org/downloads/buildroot-"${BUILDROOT_VERSION}".tar.gz | tar -xzf - -C src --strip-components=1
+    TARBALL=buildroot-${BUILDROOT_VERSION}.tar.gz
+    rm -rf "${TARBALL}"
+    curl -LO https://buildroot.org/downloads/"${TARBALL}"
+    SHA256=31a0dd7f5df75d20f6b74f0684918b643e5bf08d87dd5f9f02525852eecccf95
```

Comment:
add it as a file, then use `sha256sum -c <file>`
see: https://shapeshed.com/unix-sha1sum/

Reply:
Sure